### PR TITLE
send logic moved to drivers

### PR DESCRIPTION
**Pull request by llabbasmkhll**
hi. thanks for your excellent package.

i moved send logic from TelegramChannel file to Drivers. this way we have clear and more focused code in the channel file.

### CODE REVIEW

**Comment by irazasyed** [comment]
Interesting. Could you please fix the tests?

**Comment by llabbasmkhll** [comment]
I guess the tests are fixed.
i added [laravel testbench](https://packages.tools) to the package and I learned a lot throwout the way.

File: src/TelegramBase.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace NotificationChannels\Telegram;
+
+use JsonSerializable;
+use NotificationChannels\Telegram\Traits\HasSharedLogic;
+
+/**
+ * Class TelegramMessage.
```
Comment (irazasyed) [inline src/TelegramBase.php:9]: Please revise the class name here. Thanks!

Comment (llabbasmkhll) [reply src/TelegramBase.php]: oh ok done

**Comment by irazasyed** [comment]
Thank you!

**Pull request merged**